hal/src/nRF52840/usb_hal_cdc.c

```diff
@@ -417,7 +458,7 @@ uint32_t usb_uart_get_baudrate(void) {
 }
 
 void usb_hal_attach(void) {
-    if (m_usb_instance.state == HAL_USB_STATE_DETACHED) {
+    if (m_usb_instance.state <= HAL_USB_STATE_DISABLED) {
```

Comment:
If the USB device is attached then return, so it should be `m_usb_instance.state >= HAL_USB_STATE_ATTACHED`

Reply:
`>= HAL_USB_STATE_DETACHED`, right?

Reply:
`>= HAL_USB_STATE_ATTACHED`, the value of state larger than HAL_USB_STATE_ATTACHED is attached, no need to attach again